### PR TITLE
docs: mention the correct Postgres version to use

### PR DESCRIPTION
**Pull request by davidbgk**
When using version 13.2, one test (test_create_service_and_history_is_transactional) is failing because the `IntegrityError` message slightly differs from the one using version 12.6 of the database.

See #1217 for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ source  ~/.pyenv/versions/3.9.1/bin/virtualenvwrapper.sh
 
 `workon notifications-api`
 
-9. Install [Postgres.app](http://postgresapp.com/).
+9. Install [Postgres.app](http://postgresapp.com/) (version 12.6, not the latest).
```
Comment (AntoineAugusti) [inline README.md:60]: We run 11.9 in staging/production at the moment. May be good to update the version here while we're at it! https://github.com/cds-snc/notification-api/blob/0ede6a61b48289236d1873124965d2bc22a9b27b/docker-compose.yml#L5

File: README.md [hunk 2]
```diff
@@ -196,3 +198,8 @@ __Problem__: Messages are in the queue but not sending
 
 __Solution__: Check that `celery` is running. 
 
+---
+
+__Problem__: Running tests, `test_create_service_and_history_is_transactional` is failing with this error: `assert 'column "name" violates not-null constraint' in '(psycopg2.errors.NotNullViolation) null value in column "name" of relation "services_history" violates not-null constraint`
+
+__Solution__: Ensure your PostgreSQL version is 12.6, this error happens with version 13+.
```
Comment (AntoineAugusti) [inline README.md:205]:
```suggestion
__Solution__: Ensure your PostgreSQL version is 11.9, this error happens with version 13+.
```

**Review by AntoineAugusti — CHANGES_REQUESTED** [review]
Thanks for opening a PR!

File: README.md (hunk 2 above)
Comment (davidbgk) [reply README.md:205]: I would like to emphasis the fact that 13+ is not supported yet, but maybe we could provide a range for that particular known error, something like: `Ensure your PostgreSQL version is above 11 but below 13.`

File: README.md (hunk 1 above)
Comment (davidbgk) [reply README.md:60]: The Postgres app [currently allows](https://postgresapp.com/downloads.html) 13.2, 12.6, 11.11, 10.16 and older ones. Given that version 12.6 is the upper one which still works I would recommend it in the install process for the dev environment. We might suggest 11.11 to be closer to the staging/prod env.

**Comment by sastels** [comment]
@davidbgk @jimleroyer is this still relevant (looks like it is?) / can we merge or close?

**Comment by davidbgk** [comment]
> @davidbgk @jimleroyer is this still relevant (looks like it is?) / can we merge or close?

I still think it's pertinent for a newcomer, feel free to amend the commits if you need to!

**Pull request closed without merging**